tests/format/misc/errors/js/smart-pipeline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -55,7 +55,7 @@ exports[`snippet: #8 [babel] format 1`] = `
 `;
 
 exports[`snippet: #9 [babel] format 1`] = `
-"Can not use 'yield' as identifier inside a generator. (1:39)
+"Unexpected yield after pipeline body; any yield expression acting as Hack-style pipe body must be parenthesized due to its loose operator precedence. (1:39)
```

Comment:
I don't know well about this. I'll investigate.

Reply:
I think it's fine to update the error message.